Meta/build-image-limine.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/sh
+
+set -e
+
+die() {
+    echo "die: $*"
+    exit 1
+}
+
+if [ ! -d "limine" ]; then
+    echo "limine not found, the script will now build it"
+    git clone --depth 1 --branch v2.74.6 --single-branch https://github.com/limine-bootloader/limine
```

Comment:
Damn, Limine moves fast. The newest version is already on 2.78.2. Looks to be [quite a bit](https://github.com/limine-bootloader/limine/compare/v2.74.6...v2.78.2) of changes. Hopefully no breaking changes.

Reply:
Yeah, Limine updates very frequently, but I don't remember the last time there were any breaking changes. I picked the branch v2.74.6 due to the fact it's way simpler to set up because in later version the author switched from using a Makefile to autoconf which I'm not familiar with. Should I update it? If so, should I stick it to master or some specific version?

Reply:
I updated Limine to v2.78.2 in f1039a93c5d550cc51251e058a0e67890442a49d, I thought about it for a while and using the master branch would be quite irresponsible as really any change could break it very easily.